### ISSUE TITLE
A fix for the jittery mesh prediction

### ISSUE THREAD

**Issue by MatthM31** [Pixel-Talk/PEAR#15]
Hi, I think I found the cause of mesh jitters during prediction.

https://github.com/Pixel-Talk/PEAR/blob/be57bffdebdd16d57c0faad4754cb5bbade99fa0/inference_images.py#L270
https://github.com/Pixel-Talk/PEAR/blob/be57bffdebdd16d57c0faad4754cb5bbade99fa0/configs/infer.yaml#L12

The EHM model's BACKBONE has a dropout active throughout the inference code.

Adding a simple `ehm_model.eval()` fixes the jitter.

Here is the before and after:

No eval:
https://github.com/user-attachments/assets/f788ef64-0d2d-4ac1-add1-422a995994b6

With eval:
https://github.com/user-attachments/assets/04860a43-78db-47fb-8dd8-fc1a8c999ca2

**Comment by WuJH2001** [comment]
Hi @MatthM31, thank you very much for pointing this out.
